src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -201,13 +201,8 @@ PMComplexTest >> testComplexCollection [
 
 { #category : #tests }
 PMComplexTest >> testComplexConjugate [
-	| z complexConjugateOfZ expected |
```

Comment:
Consider giving the category a better name, perhaps `arithmetic` (is complexConjugate in that category?).

Reply:
Yes, it's a good idea to have descriptive categories
I will merge this one and then we will fix all categories in PMComplex